doc/source/conf.py

```diff
@@ -71,6 +71,11 @@
         "threshold": 0.2,
         "limit": 7,
         "minMatchCharLength": 3,
+        "ignoreLocation": True,
+        "index_patterns": {
+            "examples/api/*": "ALL_NODES",
+            "examples/sphinx_examples/*": "TITLES + PARAGRAPHS",
```

Comment:
These shouldn't be strings but aliases implemented in our `__init__.py` file.

```python
PARAGRAPHS = [docutils.nodes.paragraph]
```

Thus, when users use them, we just get a long list of docutils nodes:

```python
TILES + PARAGRAPHS
```